### PR TITLE
Add list:basic:integer: field

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: resolwe/flow/static/flow/typeSchema.json [hunk 1]
```diff
@@ -241,19 +241,19 @@
...
-        "list_data": {
+        "list_integer": {
             "type": "object",
             "required": ["type", "value"],
             "additionalProperties": false,
             "properties": {
                 "type": {
                     "type": "string",
-                    "pattern": "^list:data:[a-z0-9:]+:$"
+                    "pattern": "^list:(basic:integer|data:[a-z0-9:]+):$"
```
Comment (kostko) [inline resolwe/flow/static/flow/typeSchema.json]: Why is the `data:[a-z0-9:]+:` (without `list:`) present in both `string` and `integer` types, but for lists, it is only present under `list_integer` and not `list_string`? This seems inconsistent.

**Comment by kostko** [comment]
LGTM

File: resolwe/flow/static/flow/typeSchema.json (hunk 1 above)
Comment (dblenkus) [inline resolwe/flow/static/flow/typeSchema.json]: Thanks for pointing this out. It is left-over from the old platform, where ids were uuids. I've fixed it and added test

Comment (kostko) [inline resolwe/flow/static/flow/typeSchema.json]: Yes, I've seen the fix and the test, everything looks good now.

**Pull request merged**